autoload/coqtail/panels.vim

```diff
@@ -271,3 +274,13 @@ function! coqtail#panels#cleanup() abort
 
   call s:clearhl(winnr())
 endfunction
+
+" Getter for variables local to the main buffer
+function! coqtail#panels#getvar(var) abort
+  return getbufvar(b:coqtail_panel_bufs.main, a:var)
+endfunction
```

Comment:
I like these functions, it should make it easier if we decide to switch to some other way of sharing variables between the panels.